### DISCUSSION TITLE
Question: Initial setup & disable bluetooth when in local api mode?

### DISCUSSION THREAD

**Discussion by DirkBaumeister** [SonnenladenGmbH/APsystems-EZ1-API-HomeAssistant#25] [Q&A]
Hi,
I am currently investigating what inverter would fit best for my home setup.
Regarding this I have two questions:

- When I do the initial setup (As stated in the note in this repo), do I have to connect the ez1 to the cloud to get it into my wifi? I have a separate IoT Wifi without internet access, so this would not be ideal. Or can I connect the ez1 to this wifi without the need to connect it to the cloud?
- I have already seen that there seems to be a bluetooth "lockdown mode" when setup in cloud configuration to disable bluetooth. Can I disable bluetooth also when I am operating the ez1 in local api mode only? Background: I don't want my inverter to be available to "the outside world" to be connected in any kind and only accessible via my IoT wifi.

Thanks in advance!

**Comment by mawoka-myblock** [comment]
As far as I know, both is possible.

**Comment by DirkBaumeister** [comment]
Thank you :) Would it be possible for anyone to verify it? I am currently deciding what inverter I want to buy and these two questions would be essential for my decision. Thank you again!

**Reply by mawoka-myblock** [reply]
Either @SonnenladenGmbH has to check this or I may be able to verify it in some days.

**Reply by DirkBaumeister** [reply]
Thank you in advance! :)

**Reply by DirkBaumeister** [reply]
Sooo.. Any chance to check the above questions? :)

**Reply by mawoka-myblock** [reply]
Nah, unfortunately not. I also don't have the connection, it's only @SonnenladenGmbH. Gonna discuss that, but if will still take at least another month, probably 2 or 3.

**Reply by DirkBaumeister** [reply]
Yeah, I am aware of this :) I meant if anyone had the chance to check if bluetooth can be disabled by the app if in local mode :)

**Comment by DirkBaumeister** [comment]
Good evening,
soooo.. @SonnenladenGmbH may I ask if there is any chance that this could be part of a future firmware update? Any new updates so far? :)
